### PR TITLE
MadaraReader: add source OlympusScanlation

### LINKED ISSUE #8019: Olympus Scanlation / Martial Manga: Cambio de página
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### Describe your suggested feature

Olympus scanlation(Martial manga) a cambiado de página lo cual hace que ya no se pueda leer en martialmanga.com ahora su web se encuentra alojada en https://olympusscanlation.com/ actualizarlo por favor 

### Other details

_No response_

### PR DESCRIPTION
**Pull request by wolfghost9898**
Closes #8019

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by alessandrojean** [comment]
> Referenced all related issues in the PR body (e.g. "Closes #xyz")

Closes #8019

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 1]
```diff
@@ -314,6 +314,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Novels Town", "https://novelstown.cyou", "ar"),
         SingleLang("Oh No Manga", "https://ohnomanga.com", "en", isNsfw = true),
         SingleLang("Olho da Lua", "https://olhodalua.xyz", "pt-BR", className = "OlhoDaLua", overrideVersionCode = 1),
+        SingleLang("Olympus Scanlation","https://olympusscanlation.com/","es"),
```
Comment (jopejoe1) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:317]: The `/` at the end of the URL is redundant.

File: multisrc/overrides/madara/olympusscanlation/src/OlympusScanlation.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package eu.kanade.tachiyomi.extension.es.olympusscanlation
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+
+class OlympusScanlation : Madara("Olympus Scanlation", "https://olympusscanlation.com/", "es")
```
Comment (jopejoe1) [inline multisrc/overrides/madara/olympusscanlation/src/OlympusScanlation.kt:5]: This file is redundant as this source does not have any changes from the base madara source.

**Comment by jopejoe1** [comment]
Please add this icon too
[ic_launcher(3).zip](https://github.com/tachiyomiorg/tachiyomi-extensions/files/7760571/ic_launcher.3.zip)

**Comment by arkon** [comment]
Should probably remove Martial Manga from WPMangaReaderGenerator too, if I'm understanding #8019 correctly.

**Pull request closed without merging**